### PR TITLE
Pipelines install feedback

### PR DESCRIPTION
**Pull request by josh-padnick**
I tried setting up Pipelines on my own and wrote out these updates to better answer my own questions.

### CODE REVIEW

File: _docs-sources/pipelines/hello-world/index.md [hunk 1]
```diff
@@ -24,14 +24,18 @@ Before you begin, make sure you have:
 - [Terragrunt](https://terragrunt.gruntwork.io/) installed on your system
 - A [classic GitHub PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#personal-access-tokens-classic) with `repo` scopes and access to Gruntwork modules
 
+:::info
+To create a classic GitHub PAT, go to https://github.com/settings/profile, click on `Developer Settings`, then `Personal access tokens`, then `Tokens (classic)`, then `Generate new token (classic)`. In the "Note" field, enter "Gruntwork Pipelines POC" (or something similar), select the `repo` scope checkbox, and click `Generate token`. Keep your token handy; we'll be using it shortly.
```
Comment (MoonMoon1919) [inline _docs-sources/pipelines/hello-world/index.md:28]:
```suggestion
To create a classic GitHub PAT, go to https://github.com/settings/profile, click on `Developer Settings`, then `Personal access tokens`, then `Tokens (classic)`, then `Generate new token (classic)`. In the "Note" field, enter "Gruntwork Pipelines POC" (or something similar), select the `repo` and `workflow` scope checkboxes, then click `Generate token`. Keep your token handy; we'll be using it shortly.
```

Comment (MoonMoon1919) [reply _docs-sources/pipelines/hello-world/index.md:28]: Sorry for missing this before!

**Comment by josh-padnick** [comment]
@ellisonc @MoonMoon1919 This is ready to go, however there's a merge conflict and I'm nervous about merging this now without coordinating with you. Would one of you mind taking the lead on this PR from here?

**Comment by MoonMoon1919** [comment]
> @ellisonc @MoonMoon1919 This is ready to go, however there's a merge conflict and I'm nervous about merging this now without coordinating with you. Would one of you mind taking the lead on this PR from here?

No problem, will get it over the line

**Review by ellisonc — APPROVED** [review]

**Comment by josh-padnick** [comment]
Thanks! FYI, I just kicked off a new release.

**Pull request merged**